File: source/retrofitjava/app/src/main/java/com/zero/retrofitjava/MainActivity.java
package com.zero.retrofitjava;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        RetrofitDemo retrofitDemo = new RetrofitDemo();
        retrofitDemo.listRepos();
        OkHttpDemo okHttpDemo = new OkHttpDemo();
        okHttpDemo.listReposEx();

    }



}

File: source/jetPackDemo/app/src/main/java/com/zero/jetpackdemo/MainActivity.kt
package com.zero.jetpackdemo

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.zero.jetpackdemo.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Log.i("Zero","onCreate")
    }

    override fun onResume() {
        super.onResume()
        Log.i("Zero","onResume")
    }
}
